recipes/fiji-ilastik/meta.yaml

```diff
@@ -0,0 +1,21 @@
+{% set name = "fiji-ilastik" %}
+{% set version = "1.8.2" %}
+ 
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  url: https://maven.scijava.org/service/local/repositories/releases/content/org/ilastik/ilastik4ij/{{ version }}/ilastik4ij-{{ version }}.jar
+  sha256: 7ed05f4941d6d870e70bae3ff7e183a1d750ea5ed63f0efaf6bcbac500c715a0
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  host:
+    - fiji >=20170530
```

Comment:
out of curiosity, is this some convention to add fiji here? I mean there's nothing to be built in this package, so I'd think one could get away with having run requirements only.